.github/workflows/lighthouse-prod.yml

```
@@ -113,6 +113,27 @@ jobs:
             echo "EOF"
           } >> "$GITHUB_OUTPUT"
 
+      - name: Run Lighthouse CI (Mobile, against live deployment)
+        if: steps.loop_guard.outputs.skip != 'true'
+        uses: treosh/lighthouse-ci-action@v12
+        with:
+          urls: ${{ steps.urls.outputs.list }}
+          configPath: apps/harrychang-me/.lighthouserc.mobile.json
+          artifactsDirectory: .lighthouseci-mobile
```

actionlint (1.7.12)

[error] input "artifactsDirectory" is not defined in action "treosh/lighthouse-ci-action@v12". available inputs are "artifactName", "basicAuthPassword", "basicAuthUsername", "budgetPath", "configPath", "runs", "serverBaseUrl", "serverToken", "temporaryPublicStorage", "uploadArtifacts", "uploadExtraArgs", "urls"

(action)